static/new_home/style.css

```diff
@@ -631,6 +631,58 @@ header
   color: #eb5d1e;
 }
 
+@media screen and (min-width: 768px)
+  {
+    .contact-us-grid
+    {
+      display: grid;
+    }
+    .contact-us-bottom-card{
+      margin-left: -10px;
+    }
+  }
+
+  @media screen and (max-width: 767px)
+  {
+    .contact-us-bottom-card
+    {
+      margin-top: -70px;
+    }
+    
+  }
+
+  @media (max-width: 1200px) and (min-width: 768px)
+  {
+    .contact-us-margins
+    {
+      margin: 15px 15px;
+    }
+    .containerly
+    {
+      width: 90%;
+    }
+    .contact-us-logos{
+      padding-left: 0rem !important;
+    }
+    .contact-us-socials{
+      padding-top: 0rem !important;
+      margin-left: -1.5rem !important;
+      margin-right: -1.5rem !important;
```

Comment:
Instead of using negative margins, remove some positive margins or padding.
(Can be done using CSS classes itself)